packages/fxa-settings/src/components/Settings/DataBlock/index.tsx

```diff
@@ -4,7 +4,10 @@
 
```

Comment:
Wondering if `DataBlock` and `GetDataTrio` should be moved up directly into `/components` and out of `settings` since they are now used in multiple contexts. 🤔